geonode/documents/views.py

```diff
@@ -489,18 +490,57 @@ def document_metadata_detail(request, docid, template='documents/document_metada
 @login_required
 def document_list(request):
     documents = Document.objects.all()
+	filename = (str(datetime.datetime.now()) + "_documents.csv")
```

Comment:
I think you can also remove parenthesis around string definition : `filename = str(datetime.datetime.now()) + "_documents.csv"` should work